WordPress/src/main/java/org/wordpress/android/ui/pages/PagesFragment.kt

```diff
@@ -348,7 +349,7 @@ class PagesFragment : Fragment(), ScrollableViewInitializedListener {
 
         viewModel.createNewPage.observe(viewLifecycleOwner, Observer {
             if (modalLayoutPickerFeatureConfig.isEnabled()) {
-                mlpViewModel.show()
+                mlpViewModel.show(SupportedBlocks.fromAssets(requireContext()))
```

Comment:
You are right on this @malinajirka 👍 
Probably I would call it something like `createPageFlowTriggered` since I already have a method `onCreatePageClicked` (referring to the internal create page button) in the VM. If that's ok I will keep a note on this (to refactor later) to avoid unnecessary conflicts since this is part of a chain of 5 Android PRs at this point.

Reply:
👍 SGTM! Thanks

Reply:
Since I touched the show method for the provider extraction bellow I also fixed this with 27a829e3e5146e45b7c4a31da9baac59825255bb :)